src/main/java/ch/njol/skript/effects/EffTitle.java

```diff
@@ -0,0 +1,103 @@
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.Kleenean;
+
+@Name("Title")
+@Description("Sends a title/subtitle to the given player(s) with optional fadein/stay/fadeout times for Minecraft versions 1.11 and above.")
+@Examples({"send title \"Competition Started\" with subtitle \"Have fun, Stay safe!\" to player for 5 seconds",
+		"send title \"Hi %player%\" to player", "send title \"Loot Drop\" with subtitle \"starts in 3 minutes\" to all players",
+		"send title \"Hello %player%!\" with subtitle \"Welcome to our server\" to player with fadein 1 second for 5 seconds with fadeout 1 second"})
+@Since("INSERT VERSION")
+public class EffTitle extends Effect {
+	
+	static {
+		if (Skript.isRunningMinecraft(1, 11)) {
+			Skript.registerEffect(EffTitle.class, "send title %string% [with subtitle %-string%] to %players% [with fade[(-| )]in %-timespan%] [for %-timespan%] [with fade[(-| )]out %-timespan%]");
+		} else {
+			Skript.registerEffect(EffTitle.class, "send title %string% [with subtitle %-string%] to %players%");
+		}
+	}
+	
+	@SuppressWarnings("null")
+	private Expression<String> title, subtitle;
+	@SuppressWarnings("null")
+	private Expression<Player> recipients;
+	@SuppressWarnings("null")
+	private Expression<Timespan> fadein, stay, fadeout;
+	
+	@SuppressWarnings({"unchecked", "null"})
+	@Override
+	public boolean init(final Expression<?>[] exprs, final int matchedPattern, final Kleenean isDelayed, final ParseResult parser) {
+		title = (Expression<String>) exprs[0];
+		subtitle = (Expression<String>) exprs[1];
+		recipients = (Expression<Player>) exprs[2];
+		if (Skript.isRunningMinecraft(1, 11)) {
+			fadein = (Expression<Timespan>) exprs[3];
+			stay = (Expression<Timespan>) exprs[4];
+			fadeout = (Expression<Timespan>) exprs[5];
+			return true;
+		}
+		return true;
+	}
+	
+	@SuppressWarnings("null")
+	@Override
+	protected void execute(final Event e) {
+		String title = this.title.getSingle(e);
+		String subtitle = this.subtitle != null ? this.subtitle.getSingle(e) : null;
+		if (Skript.isRunningMinecraft(1, 11)) {
```

Comment:
still wondering why this isn't a method check like other syntaxes with special requirements

Reply:
I would definitely love to attempt that, but that part has me confused a bit. 
I will do my best to see what I can figure out.
If anyone wants to offer some guidance, I am open to any help!

Reply:
Okay @Pikachu920  I think I managed to do it properly this time. Hopefully this is what you were meaning :)